config.toml

```diff
@@ -1,5 +1,6 @@
 # The URL the site will be built for
-base_url = "https://rust-basel.ch"
+#base_url = "https://rust-basel.ch"
```

Comment:
I think this still myst be `base_url = "https://rust-basel.ch"`

Reply:
Fixed in 8835975daec0710a4b4b16d5cabed44768fbadd0